src/main/java/com/pikachu/purple/infrastructure/persistence/perfume/entity/PerfumeBrandJpaEntity.java

```diff
@@ -0,0 +1,40 @@
+package com.pikachu.purple.infrastructure.persistence.perfume.entity;
+
+import com.pikachu.purple.domain.perfume.PerfumeBrand;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Entity
+@Table(name = "perfume_brand")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class PerfumeBrandJpaEntity {
+
+    @Id
+    @Column(name = "Brand_name")
```

Comment:
p4) 요것도 같이 수정되면 좋겠네요!
물론 DB의 column이 대소문자를 구별하지는 않지만, 일반적인 프로그래밍 언어 관점에서 `_`를 통해 구분하는 snake case를 가정할 때, 변수명의 첫 시작이 소문자인 점을 감안했을 때 약간 부자연스러운 것 같아요!
```suggestion
    @Column(name = "brand_name")
```

Reply:
entity 전체적으로 반영 완료했습니다!